packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -193,6 +193,18 @@ class EditorPage {
 		return elements[ elements.length - 1 ];
 	}
 
+	// iOS might return the element without scrolling.
+	// It will depend on how fast the list renders, e.g on CI.
```

Comment:
Not sure I fully understand the explanation of this comment, what will depend on how fast the list renders?

Reply:
Sorry about that 😅 I will rephrase it better

Reply:
I've updated in https://github.com/WordPress/gutenberg/pull/47814/commits/3ff8caf164183c72d1b0e75e4e578929e0759112 let me know if it's more clear now or if it needs more clarification, thank you!

Reply:
Thanks for elaborating! So, for iOS, is it possible that the test succeeds without actually scrolling to the bottom? Could that produce false positives if we don't wait for all blocks to be rendered?

Reply:
> I don't think so, if anything we would be keeping the same functionality we've had because previously for iOS we didn't scroll at all, we would just get the last block without waiting if others have finished rendering.

Ah true, I didn't remember that on iOS we didn't scroll 😅.

> I do think these tests need revisiting maybe we want to test something different than just setting all blocks we support, initially I think we just did this to see if the editor would crash with any changes.

I agree, now with the integration tests I think we can mostly cover all crashes that we intended to do with these E2E tests. Eventually, we could consider removing these ones, or even better, incorporating the ones we do in GB-mobile.

Reply:
>I agree, now with the integration tests I think we can mostly cover all crashes that we intended to do with these E2E tests. Eventually, we could consider removing these ones, or even better, incorporating the ones we do in GB-mobile.

Yup although the recent crash we had was only happening with the E2E tests, integration tests were working, so definitely we need to have both options that would complement each other 😄